ew/backend/user.py

```diff
@@ -273,6 +279,8 @@ def __init__(self, ew_id = None, member = None, id_user = None, id_server = None
                     self.time_lasthit = result[55]
                     self.verified = result[56]
                     self.gender = result[57]
+                    self.party_points = result[58]
```

Comment:
we can leave this for now, but when it comes to temporary events, using stats instead of adding to the user definition is the better option